packages/amplify-graphql-transformer-core/src/__tests__/utils/directive-wrapper.test.ts

```diff
@@ -186,5 +186,31 @@ describe('Transformer Core Util Tests', () => {
       expect(newArgs.timestamps.createdAt).toEqual('createdOn');
       expect(newArgs.timestamps.updatedAt).toEqual('updatedOn');
     });
+
+    it(': Should skip location when deep cloning', () => {
```

Comment:
Nit: Leading ": " diverges from the naming pattern.

Reply:
I took it from here https://github.com/aws-amplify/amplify-category-api/blob/267419a2090a6c1feb8a3529b50b01b78734385b/packages/amplify-graphql-transformer-core/src/__tests__/utils/directive-wrapper.test.ts#L50 .

Which means there are two patterns in this file :(

Reply:
Gotcha. Missed that.